src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ListViewModel.cs

```diff
@@ -14,10 +14,14 @@ namespace Microsoft.CmdPal.UI.ViewModels;
 
 public partial class ListViewModel : PageViewModel
 {
+    private readonly HashSet<ListItemViewModel> _itemCache = [];
+
+    // TODO: Do we want a base "ItemsPageViewModel" for anything that's going to have items?
```

Comment:
Pretty sure this the ListPage is the only one that this would apply to

(though, there is the context menu which we'd also love to make filterable. Not really a page, but still "a list of commands that can be filtered")